src/service/api/balance.ts

```diff
@@ -0,0 +1,27 @@
+import {ApiPromise} from '@polkadot/api';
+import {BN, BN_TEN} from '@polkadot/util';
+
+/**
+ * turns a string to a Balance BN in order to send through API or
+ * show the formatted version to user
+ * @param api
+ * @param input String (user input E.g. 2.33 or 20 or 0.33)
+ * @returns BN
+ */
+export function getBalanceFromString(api: ApiPromise, input: string): BN {
```

Comment:
Should we move this file to the api module? like `src/api/utils/balance.ts`

Reply:
got auto-merged. just made a new pr for this 👍